### PR TITLE
fix(pci.project): incorrect value for the gateway

### PR DESCRIPTION
**Pull request by vks-ovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/l3-services` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-10019
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~~[ ] Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

## Description

Addition of gateway name and gateway ip for private network list page.

### CODE REVIEW

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/private-networks/private-networks.controller.js [hunk 1]
```diff
@@ -5,15 +5,89 @@ const CONTAINER_NAME = 'pci.projects.project.privateNetwork';
...
+    this.PciPrivateNetworks = PciPrivateNetworks;
   }
 
   $onInit() {
     this.loadMessages();
+    this.isLoading = false;
     this.criteria = getCriteria('id', this.networkId);
+    this.availableNetworks = null;
+    this.gatewaySubnetObj = null;
+    this.networksSubnet();
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/private-networks.controller.js:22]: **nitpick:** Please prefix the call with verb.
```suggestion
    this.getNetworksSubnet();
```

File: packages/manager/modules/pci/src/projects/project/private-networks/private-networks.routing.js [hunk 2]
```diff
@@ -70,6 +70,22 @@ export default /* @ngInject */ ($stateProvider) => {
...
+        return $q
+          .all(
+            privateNetworksRegions.map((region) =>
+              PciPrivateNetworks.getGateways(projectId, region)
+                .then((data) => data)
+                .catch(() => []),
+            ),
+          )
+          .then((gatewaysData) => gatewaysData.flat());
+      },
```
Comment (anooparveti) [inline packages/manager/modules/pci/src/projects/project/private-networks/private-networks.routing.js:88]: **nitpick:** Use the regionalised API `/cloud/project/{serviceName}/aggregated/gateway` and replce multiple calls with the single one.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/private-networks/private-networks.controller.js (hunk 1 above)
Comment (vks-ovh) [reply packages/manager/modules/pci/src/projects/project/private-networks/private-networks.controller.js]: Taken care.

File: packages/manager/modules/pci/src/projects/project/private-networks/private-networks.routing.js (hunk 2 above)
Comment (vks-ovh) [reply packages/manager/modules/pci/src/projects/project/private-networks/private-networks.routing.js]: Taken care.

**Review by anooparveti — APPROVED** [review]

**Pull request merged**